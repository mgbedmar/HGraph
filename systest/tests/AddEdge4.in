1
1
Donald Knuth
2
1
1
applying markov chains
1
1
2
swarm algorithms
1
2
0
1
2
0
2
4
