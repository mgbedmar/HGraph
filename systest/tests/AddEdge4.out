S'ha afegit l'aresta que va de 1 Donald Knuth author al node 1 applying markov chains paper
S'ha afegit l'aresta que va de 1 Donald Knuth author al node 2 swarm algorithms paper
Tenim el següent node :1del tipo authory su contenido es Donald Knuth
La seva llista d'adjacències és 
Id intern 	 IdClase	Contingut	Tipus
0		 1	applying markov chains	 paper
1		 2	swarm algorithms	 paper
No té més adjacents
Tenim el següent node :1del tipo papery su contenido es applying markov chains
Tenim el següent node :2del tipo papery su contenido es swarm algorithms
